README.md

```diff
@@ -155,7 +155,10 @@ Ruff can also be used as a [pre-commit](https://pre-commit.com/) hook via [`ruff
   hooks:
     # Run the linter.
     - id: ruff
-      args: [ --fix ]
+      entry: ruff check
+      args:
+        - --force-exclude
```

Comment:
Is this part necessary given that `--force-exclude` is in the entry? https://github.com/astral-sh/ruff-pre-commit/blob/522df5be28c6c515e78ef3c2790b9f233680f04d/.pre-commit-hooks.yaml#L4

Reply:
I was just copying the pre-commit config. Your right it probably isn't. I'll remove the arg.